course/templates/course/gradebook-participant-list.html

```diff
@@ -4,15 +4,15 @@
 {% load static %}
 
 {% block title %}
-  {% trans "List of Participants" %} - {{ relate_site_name }}
+  {% trans "List of participants" %} - {{ relate_site_name }}
```

Comment:
Why? Other page titles use [title case](https://en.wikipedia.org/wiki/Title_case), why is that not OK here?

Reply:
Sure, I'll revert that. (I was thinking not to re-translate just because a capitalized letter).